javalib/src/main/scala/java/lang/StackTraceElement.scala

```diff
@@ -0,0 +1,17 @@
+package java.lang
+
+final class StackTraceElement private[lang] (private val symbol: String) {
+  override def toString: String =
+    symbol
+
+  override def hashCode: scala.Int =
+    symbol.hashCode
+
+  override def equals(that: Any): scala.Boolean =
+    that match {
+      case that: StackTraceElement =>
+        symbol == that.symbol
+      case _ =>
+        false
+    }
+}
```

Comment:
These APIs are not really sufficient for typical usage. We'd at least want to also support:
```
@java.lang.StackTraceElement::getClassName_class.java.lang.String
@java.lang.StackTraceElement::getMethodName_class.java.lang.String
@java.lang.StackTraceElement::isNativeMethod_bool
```